libvast/src/arrow_extension_types.cpp

```diff
@@ -111,18 +111,72 @@ std::string address_extension_type::Serialize() const {
   return id;
 }
 
+const std::string address_extension_type::id = "vast.address";
+
+const std::shared_ptr<arrow::DataType> address_extension_type::arrow_type
+  = arrow::fixed_size_binary(16);
```

Comment:
Why is this addition part of this commit?

Reply:
I learned that I want to unify the shape of these structs, e.g. to be able to make the write a generic round-trip test for all these. So I added this field when adding the second extension type to be able to write that code.

Reply:
@tobim: is there anything else stopping us from going forward with this?

Reply:
No, apologies for the delay.